.github/workflows/daily-build.yml

```diff
@@ -18,25 +21,38 @@ jobs:
           distribution: 'adopt'
           java-version: 11
 
+      - name: Set environment variable
+        if: github.event.action == 'check_connector_for_breaking_changes'
+        run: |
+          echo "CHECK_BRAEKING_CHANGES=true" >> $GITHUB_ENV
+          echo "GRADLE_SKIP_TASKS=-x :oracledb-compiler-plugin-tests:test"  >> $GITHUB_ENV
```

Comment:
Why gradle plugin tests are skipped in this case?

Reply:
The compiler plugin uses the intermediate pack built inside the libraries, we need to find a way to use the docker image inside these tests. As of now we are not checking for the compiler plugin test cases thus skipped.

Reply:
Ack